src/includes/_judgment_text.scss

```diff
@@ -295,11 +295,13 @@
 .judgment-body {
   table {
     table-layout: auto;
-    width: 100%;
-    margin: 1rem auto;
+    width: calc(100% + 53px);
```

Comment:
This 53px seems magical, should this be a constant somewhere?
Is this constant fine on both mobile and desktop?

Reply:
This doesn't feel right on mobile widths:
<img width="478" alt="image" src="https://github.com/user-attachments/assets/1d031cb3-212b-4b33-9611-a7e9393d7de4">